src/assertions.ts

```diff
@@ -313,6 +313,7 @@ export async function runAssertion({
     vars: test.vars || {},
     test,
     logProbs,
+    ...(assertion.config ? { config: assertion.config } : {}),
```

Comment:
If you're feeling inspired, please consider adding a test to `assertions.test.ts`

Reply:
Also done :)